sdks/python/apache_beam/io/gcp/bigtableio_read_it_test.py

```diff
@@ -0,0 +1,159 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
Can you try triggering this for a large dataset to make sure that there's no severe perf regression ?

Reply:
So far it's been tested for tables containing up to 10^10 rows, which must be sufficient for most purposes. The read times increase roughly linearly with the table size. The total read times for 10^10 rows are just under 3 hours with a high degree of consistency. That is under maxed out parallelization for 'us-central1', or 600 workers total. It seems quite possible to read even larger tables, as long as spending weeks, possibly months for such tests is not an issue.